lib/utils/appList.js

```diff
@@ -21,6 +21,14 @@ const MAPPINGS = {
     path: [7, 0, 3, 2, 1, 0, 2],
     fun: (price) => price === undefined ? 'FREE' : price
   },
+  currency: {
+    path: [7, 0, 3, 2, 1, 0, 2],
+    fun: (price) => price === undefined ? false : price.match(/([^0-9\.,\s]+)/)[0]
```

Comment:
why default to false instead of undefined?

Reply:
fixed